src/gas/fetchGasEstimatesViaEthFeeHistory.test.ts

```diff
@@ -51,79 +30,18 @@ describe('fetchGasEstimatesViaEthFeeHistory', () => {
   };
 
   it('calculates target fees for low, medium, and high transaction priority levels, as well as the network congestion level', async () => {
```

Comment:
remove `as well as the network congestion level`?

Reply:
Yup, good catch! Fixed in 52adcd752e2dfc5e47ba1a2af39650d90d0354b3.